src/common/types/clusters.ts

```diff
@@ -24,12 +24,19 @@ export type HostSubnet = {
   humanized: string;
 };
 export type HostSubnets = HostSubnet[];
-export type NetworkConfigurationValues = V2ClusterUpdateParams & {
-  hostSubnet?: string;
-  useRedHatDnsService?: boolean;
+export type NetworkConfigurationValues = Pick<
+  V2ClusterUpdateParams,
+  | 'clusterNetworkCidr'
+  | 'clusterNetworkHostPrefix'
+  | 'serviceNetworkCidr'
+  | 'apiVip'
+  | 'ingressVip'
+  | 'sshPublicKey'
+  | 'vipDhcpAllocation'
+  | 'networkType'
+> & {
+  hostSubnet: string;
   managedNetworkingType: 'userManaged' | 'clusterManaged';
-  enableProxy: boolean;
-  editProxy: boolean;
```

Comment:
@batzionb thanks for asking the question https://github.com/openshift-assisted/assisted-ui-lib/pull/1289#discussion_r841600296 . I actually removed these ACM-specific props and updated dependent code to make sure they do not leak to common stuff.